src/render/views/CreateEncounterView.tsx

```diff
@@ -78,6 +122,32 @@ export const CreateEncounterView: React.FC<{}> = () => {
                     <Typography>Cancel</Typography>
                 </Button>
             </Box>
+            <Observer observed={{ speedDialOpen }}>
+                {({ speedDialOpen }) => {
+                    return (
+                        <SpeedDial
+                            className={styles.fab}
+                            color="primary"
+                            ariaLabel="Add Character"
```

Comment:
Let's leave all `aria` props out for now. If we want to do an accessibility pass later we definitely can (and likely should) but I'm not sure what the right usage pattern is here so I'd rather leave it alone.